pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -587,6 +589,13 @@ The SSH username, for example `centos`.
 
 **Note:** The ssh_user is not required for the production installer.
 
+### superuser_service_account_public_key
+The superuser service account public key. The operator is advised to create a RSA key-pair and supply the generated public key in X.509 SubjectPublicKeyInfo/OpenSSL PEM public key format.
```

Comment:
Remove the first line `The superuser service account public key. `
Begin with:
`This service account....